extensions/hide_idle_time.c

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
solanum isn't completely consistent here, but this should be `um_hide_idle_time.c` to follow the scheme started in `modules/` or `umode_` to follow `umode_ctcp.c`. We've got a good thing kind of going with `sno_`, `extb_`, `m_` for command things.

I'm happy to lose this one, but I feel like we should be working toward more order instead of less.